components/listview/paging.md

```diff
@@ -12,12 +12,14 @@ position: 4
 
 The ListView component can page the entire data source automatically. You can, alternatively, hook to an event and fetch each page of data yourself.
 
-To enable paging set the `Pageable` parameter of the listview to `true`.
+To enable paging set the ListView `Pageable` parameter to `true`.
 
 You can also control the number of items rendered at once through the `PageSize` parameter (defaults to 10).
 
```

Comment:
Consider mentioning the advanced pager settings existence here, early in the article.